metricbeat/module/uwsgi/status/status_integration_test.go

```diff
@@ -32,23 +32,38 @@ import (
 func TestFetchTCP(t *testing.T) {
 	compose.EnsureUp(t, "uwsgi_tcp")
 
-	f := mbtest.NewEventsFetcher(t, getConfig("tcp"))
-	events, err := f.Fetch()
-	assert.NoError(t, err)
+	f := mbtest.NewReportingMetricSetV2Error(t, getConfig("tcp"))
+	events, errs := mbtest.ReportingFetchV2Error(f)
+	if len(errs) > 0 {
+		t.Fatalf("Expected 0 error, had %d. %v\n", len(errs), errs)
+	}
+	assert.NotEmpty(t, events)
 
-	assert.True(t, len(events) > 0)
+	t.Log(events)
 	totals := findItems(events, "total")
 	assert.Equal(t, 1, len(totals))
 }
 
+func TestData(t *testing.T) {
+	compose.EnsureUp(t, "uwsgi_http")
```

Comment:
We started to skip the `EnsureUp` in the `TestData` runs as it would boot up the environment but then on CI it skips the generation anyway. And when generating it locally, I personally always boot up the env manually.